p8e-api/src/main/kotlin/io/provenance/engine/grpc/v1/IndexGrpc.kt

```diff
@@ -204,7 +205,8 @@ class IndexGrpc(
                         definitionService.get(
                             keyPair,
```

Comment:
this appears to be the signing key pair, but I think DefinitionService is expecting the encryption key pair, right? I'm wondering if this is just slightly broken (not that it matters, as the key pair will go away once the encryption abstraction is in place, right?)

Reply:
good 👁️ , unsure if you need laser